extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -144,9 +144,39 @@ tags: { # amp-list-load-more
     name: "load-more-end"
     value: ""
     mandatory_oneof: "['load-more-button', 'load-more-failed', 'load-more-end', 'load-more-loading']"
-  }
+  },
 }
 
+# The button element variant allowed in amp-list-load-more.
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
+  requires_extension: "amp-list"
+  tag_name: "BUTTON"
+  spec_name: "amp-list-load-more button[load-more-clickable]"
+  mandatory_ancestor: "AMP-LIST-LOAD-MORE"
+  attrs: {
+    name: "disabled"
+    value: ""
+  }
+  attrs: {
+    name: "load-more-clickable"
```

Comment:
To improve error messages, consider adding:

dispatch_key: NAME_VALUE_PARENT (or just NAME_VALUE)

Otherwise, this tagspec may end up being used for showing errors for buttons that don't have "load-more-clickable" or AMP_LIST_LOAD_MORE as a parent.

Reply:
Actually, maybe just NAME if the parent requirement can't be reached.

Reply:
done!

Reply:
Let's just make this NAME_DISPATCH.

It's a minor difference.

With NAME_VALUE, if someone uses `load-more-clickable=foo` this tagspec won't be used for the error message, one of the other `button` ones will. With NAME, then this tagspec will be used any time `load-more-clickable` is set, regardless of the attribute value.

I don't think it'll matter much, since nobody should be setting the value, but it will not hurt either.